src/PageViewCard.tsx

```diff
@@ -19,7 +20,20 @@ export class PageViewCard extends React.Component<IPageViewCardProps> {
     return (
       <Card className="rh-page-views-card">
         <CardHeader>
-          <Title size="lg">Repository overview page views</Title>
+          <Flex>
+            <FlexItem>
+              <Title size="lg">Repository overview page views</Title>
+            </FlexItem>
+            <FlexItem>
+              <Tooltip
+                className="rh-page-views-card-tooltip"
+                position={TooltipPosition.top}
+                content="Displays daily views of the repository and any detail pages in the Container Catalog."
+              >
+              <InfoCircleIcon color={'#8BC1F7'}/>
```

Comment:
Is this a color from the patternfly color pallete?

Reply:
Also, please indent over one level.

Reply:
Fixed indentation and using color pallete.